src/reducers/user.test.js

```diff
@@ -0,0 +1,82 @@
+/**
+ * @file user.test.js
+ * Contains tests for user.js.
+ */
+
+import reducer from './user';
+import {
+  USER_LOG_IN,
+  USER_LOG_OUT,
+  USER_SET_ROLE,
+  USER_ROLE_EDITOR
+} from '../constants';
+
+describe('reducers->user', () => {
+  it('Should return the initial state by default', () => {
+    expect(reducer(undefined, {})).toMatchSnapshot();
```

Comment:
Using snapshots to test reducer output is really nice. Not only does it fail when breaking changes are made to a reducer, it forces you to manually validate the reducer's output every time you change it.